README.md

```diff
@@ -159,7 +159,7 @@ of [galactic-data](https://github.com/anvaka/galactic-data).
 
 ```
 git clone https://github.com/anvaka/pm
-cd pm
+cd pm/src
```

Comment:
This would be incorrect however, as `package.json` is inside `pm` folder not `pm/src`

Reply:
Ok. I had trouble getting it to work, I thought I changed to arc folder and then it worked fine.